packages/vite-plugin-cloudflare/e2e/helpers.ts

```diff
@@ -29,6 +29,12 @@ const testEnv = {
 	VITEST: undefined,
 };
 
+const strictPeerDeps = {
+	pnpm: "--strict-peer-dependencies",
+	npm: "--strict-peer-deps",
+	yarn: "",
+};
```

Comment:
```suggestion
const strictPeerDeps = {
  // yarn does not have an option for strict checks
	pnpm: "--strict-peer-dependencies",
	npm: "--strict-peer-deps",	
};
```

Reply:
Added the comment but not removed the option